variables.container.tf

```diff
@@ -1,8 +1,15 @@
 variable "containers" {
   type = map(object({
-    public_access = optional(string, "None")
-    metadata      = optional(map(string))
-    name          = string
+    public_access               = optional(string, "None")
+    metadata                    = optional(map(string))
+    name                        = string
+    defaultEncryptionScope      = optional(string)
+    denyEncryptionScopeOverride = optional(bool)
+    enableNfsV3AllSquash        = optional(bool)
```

Comment:
input parameters should be in terraform azurerm form - i.e snake_casing not camelCasing

within the module, that's where you translate it to the required form for AzAPI

e.g, default_encryption_scope

this has been a requirement for other Azure modules, such as the landing zone module.